### PR TITLE
Add Icons for configured systems vendor

### PR DESCRIPTION
**Pull request by k-chu**
Add Icons for configured systems vendor

### CODE REVIEW

File: app/models/manageiq/providers/ibm_terraform/configuration_manager/configured_system.rb [hunk 1]
```diff
@@ -22,6 +40,10 @@ def console_url
     end
   end
 
+  def label_for_vendor_icon
```
Comment (skateman) [inline app/models/manageiq/providers/ibm_terraform/configuration_manager/configured_system.rb:43]: I guess this answers my [question in the decorators repo](https://github.com/ManageIQ/manageiq-decorators/pull/37#discussion_r508280574). Having the logic on two different places can make this a little bit hard to follow. You can move the decorator into this repo and have the constant and the icon selection logic in-place. 

I linked an example in my comment of how the AutoSDE folks have done this...

Comment (k-chu) [reply app/models/manageiq/providers/ibm_terraform/configuration_manager/configured_system.rb]: Thanks @skateman. I created a provider specific decorator.

**Review by agrare — APPROVED** [review]
:+1: LGTM

**Comment by agrare** [comment]
@k-chu does this have any dependencies with https://github.com/ManageIQ/manageiq-decorators/pull/37 or could either be merged first?

**Comment by Fryguy** [comment]
I think this depends on ManageIQ/manageiq-decorators#37 for those 2 icons defined there (which are used here)

**Comment by skateman** [comment]
I think the images should be also here, similarly to AutoSDE and then the decorators PR can be closed.

**Comment by k-chu** [comment]
@agrare @skateman I moved the icons to the Provider as well. I will close the other PR. Thx

**Comment by skateman** [comment]
Changes look good, but I can't test this as I have no access to a provider...
